.github/workflows/packages.yml

```diff
@@ -0,0 +1,65 @@
+---
+name: Packages
+on:
+  pull_request:
```

Comment:
I don't understand this step. Do we upload deb and rpm on every PR opened/reopened etc?

Reply:
We save them as build artifacts in the build system, not upload them into release or something